k8s/nginx-configmap.yaml

```
@@ -0,0 +1,44 @@
+    component: nginx-proxy
+data:
+  nginx.conf: |
+    events {
+        worker_connections 1024;
+    }
+    
+    http {
+        upstream ui_backend {
+            server localhost:8080;
+        }
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)